lib/transport_web/templates/page/index.html.eex

```diff
@@ -1,22 +1,46 @@
 <h1><%= dgettext("page-index", "Publish and reuse public transport data") %></h1>
 
 <section class="hero">
-  <div class="hero__container">
-    <map class="hero__column", link-class='map__link map__link--reuse'></map>
+    <div class="hero__container">
+        <map class="hero__column", link-class='map__link map__link--reuse'></map>
 
-    <div class="hero__column">
-      <h1><%= dgettext("page-index", "What is transport.data.gouv.fr?") %></h1>
-      <markdown content='<%= dgettext("page-index", "The single entry point for publication and reuse of public transport data, allowing to make people lives better.") %>'></markdown>
+        <div class="hero__column">
+            <h1><%= dgettext("page-index", "What is transport.data.gouv.fr?") %></h1>
+            <markdown content='<%= dgettext("page-index", "The single entry point for publication and reuse of public transport data, allowing to make people lives better.") %>'></markdown>
 
-      <div class="hero__links is-centered">
-        <a class="button button--fullwidth hero__link hero__link--open" href="<%= user_path @conn, :organizations %>" role="link">
-          <i class="icon icon--open" aria-hidden="true"></i>
-          <%= dgettext("page-index", "Publish my data") %>
-        </a>
-        <a class="button button--fullwidth hero__link hero__link--reuse" href="<%= dataset_path @conn, :index %>" role="link">
-          <i class="icon icon--reuse" aria-hidden="true"></i>
-          <%= dgettext("page-index", "Reuse data") %>
-        </a>
-      </div>
-    </section>
-  </section>
+            <div class="hero__links is-centered">
+                <a class="button button--fullwidth hero__link hero__link--open" href="<%= user_path @conn, :organizations %>" role="link">
+                <i class="icon icon--open" aria-hidden="true"></i>
+                <%= dgettext("page-index", "Publish my data") %>
+                </a>
+                <a class="button button--fullwidth hero__link hero__link--reuse" href="<%= dataset_path @conn, :index %>" role="link">
+                <i class="icon icon--reuse" aria-hidden="true"></i>
+                <%= dgettext("page-index", "Reuse data") %>
+                </a>
+            </div>
+        </div>
+    </div>
+</section>
+<section class="mailing-list">
+    <div class="mailing-list__container">
+        <strong>
+            <%= dgettext("page-index", "Subscribe to the newsletter to receive update") %>
+        </strong><br/>
+        <form action="//gouv.us13.list-manage.com/subscribe/post?u=5ee8bfe0f1b073b49de06a063&amp;id=13db2e9a94" method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" class="validate" target="_blank" novalidate>
```

Comment:
Shouldn't we add ```u``` and ```id``` to an environment variable, and pass them via ```webpack```?

Reply:
I think it'll be better to add the url.